argus/backend/util/encoders.py

```diff
@@ -3,12 +3,14 @@
 from json.encoder import JSONEncoder
 from uuid import UUID
 
+from flask.json.provider import DefaultJSONProvider
 import cassandra.cqlengine.usertype as ut
 import cassandra.cqlengine.models as m
 
 
 LOGGER = logging.getLogger(__name__)
 
+# TODO: Deprecated.
```

Comment:
can't we remove it ?

Reply:
Since it's an encoder class for `json.dump(s)` and not a provider class for `flask.Flask`, I'll keep it and remove the TODO marker as it is somewhat still used by some scripts and might be useful (would probably unify the default method however)